pkg/crc/constants/constants.go

```diff
@@ -196,7 +198,7 @@ func GetDefaultCPUs(preset crcpreset.Preset) int {
 	switch preset {
 	case crcpreset.OpenShift, crcpreset.OKD:
 		return 4
-	case crcpreset.Podman:
+	case crcpreset.Podman, crcpreset.Microshift:
```

Comment:
This makes me wonder if we can add these defaults to the bundle metadata instead of hardcoding this in crc

Reply:
@cfergeau add default about preset and cpu/memory uses?

Reply:
Yes, each bundle could specify its expectations wrt to CPU/memory, rather than having an hardcoded list in crc. Some food for thought for future improvements.